app/uk/gov/hmrc/tai/service/TaxAccountService.scala

```diff
@@ -49,7 +50,10 @@ class TaxAccountService @Inject()(taxAccountConnector: TaxAccountConnector) {
     implicit hc: HeaderCarrier): Future[Either[String, Option[TaxCodeIncome]]] =
     EitherT(taxAccountConnector.taxCodeIncomes(nino, year)).map(_.find(_.employmentId.contains(employmentId))).value
 
-  def taxAccountSummary(nino: Nino, year: TaxYear)(implicit hc: HeaderCarrier): Future[TaiResponse] =
+  def taxAccountSummaryOld(nino: Nino, year: TaxYear)(implicit hc: HeaderCarrier): Future[TaiResponse] =
```

Comment:
Why do we have this old method?

Reply:
we shouldnt, good spot, I have removed them now